client/src/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,54 @@
+// import AppBar from '@material-ui/core/AppBar';
+// import Toolbar from '@material-ui/core/Toolbar';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import DashboardOutlinedIcon from '@material-ui/icons/DashboardOutlined';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import AddIcon from '@material-ui/icons/Add';
+import Avatar from '@material-ui/core/Avatar';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import React from 'react';
+import logo from '../../Images/logo.png'
+
+const DashboardHeader = (): JSX.Element => {
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
```

Comment:
looks like some linting issues - I suggest syncing with the team to make sure 
1. Everyone has linters installed (eslint & prettier)
2. Everyone has the right settings :)

Reply:
I have linters installed too. I was having issues in the beginning so I changed my .eslintrc. I'm not sure which issues you're referencing. I didn't push my .eslintrc file. Maybe I should do that.